charts/zero-notifcation-service/templates/external-secret.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.externalSecret.enabled -}}
+{{- $fullName := include "zero-notification-service.fullname" . -}}
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: {{ $fullName }}
+spec:
+  backendType: {{ .Values.externalSecret.backendType }}
+  {{- if .Values.externalSecret.dataFrom }}
+  dataFrom:
+    {{ .Values.externalSecret.dataFrom | toYaml }}
```

Comment:
You should test this, you may need to also pass the result of `toYaml` into `| indent` to make sure the indenting is okay.

Reply:
okay so apparently it works only when there's 1 value, if the list is > 2 items it needs the indent